get_asteroidcount.py

```diff
@@ -0,0 +1,57 @@
+from astroquery.mpc import MPC
+
+class AsteroidWidget():
+
+    def __init(self,
+             longitude: Optional[float] = None,
+             latitude: Optional[float] = None,
+             datetime: Optional[dt] = None):
+
+        super().__init__(longitude=longitude,
+                         latitude=latitude,
+                         datetime=datetime)
+
+    def get_data(self, hazard: bool = True,
+                    large: bool = True) -> dict:
+
+    """
+    Get the current total number of near-Earth asteroids from the IAU
+    Minor Planet Center. Additionally, get the number of asteroids that
+    identified as potentially hazardous and larger than 1 km.
+
+    Args:
+        hazard: Returning the number of potentially hazardous
+            near-Earth asteroids.
+        large: Returning the number of near-Earth asteroids that
+            are larger than 1 km wide.
+    Returns:
+        A dictionary with keys `{"hazard", "large", "total"}`
+        which holds the respective count of asteroids.
+    """
+
+    asteroid_count = dict()
+
+    # Get all potentially hazardous asteroids from MPC
+    PHA = len(MPC.query_objects('asteroid', pha=1))
```

Comment:
According to the [PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), variable names should always be lowercase: _"Function names **should be lowercase**, with words separated by underscores as necessary to improve readability. Variable names follow the same convention as function names."_

In fact, all-caps is reserved for [constants](https://www.python.org/dev/peps/pep-0008/#constants) (on a module level).